src/components/WithdrawPage.tsx

```diff
@@ -225,13 +225,7 @@ const WithdrawPage = (props: Props): ReactElement => {
             {t("withdraw")}
           </Button>
         </Box>
-        <Stack
-          direction="column"
-          flex={1}
-          spacing={4}
-          width="100%"
-          maxWidth="50%"
-        >
+        <Stack direction="column" flex={1} spacing={4} width="100%">
```

Comment:
Do you always have to specify width 100%? I feel like if parent is flexing then children elements or containers should automatically fill all width?

Reply:
You are right!